addons/bus/static/src/services/bus_service.js

```diff
@@ -98,7 +119,9 @@ export const busService = {
                 uid = false;
             }
             send("initialize_connection", {
-                websocketURL: `${params.serverURL.replace("http", "ws")}/websocket`,
+                websocketURL: `${params.serverURL.replace("http", "ws")}/websocket?version=${
+                    session.websocket_worker_version
```

Comment:
Will the session be properly filled? All cases of livechat (embed)?

Reply:
I checked for: frontend, backend, public page, support page, embed live chat. All are properly filled.